styles/config/vocabularies/SAP/accept.txt

```diff
@@ -3,6 +3,7 @@ SAP AI SDK
 AI Core
 SAP Generative AI Hub
 SAP Cloud SDK
+SAP Cloud SDK for AI
```

Comment:
👍 I would suggest that in the future, we only use `SAP Cloud SDK for AI` instead of `SAP AI SDK` as a public branding.